[lte][agw] Add service area restriction documentation (#7927)
magma/magma: docs/readmes/lte/access_service_restrictions.md

    :
}
```

## Service Area/Tracking Area Restriction

> Magma also supports a zone code based service area restriction in federated LTE deployments. To utilize this service restriction, network operators need to configure (1) HSS with up to 10 Regional Restriction Zone Codes for each subscriber and (2) Magma with mapping between zone codes and lists of tracking area codes (TACs). If HSS does not provide any zone code in Update Location Answer for a subscriber, then no tracking area restriction is applied for the subscriber as the defauly behavior. On the other hand, if HSS provides a list of zone codes in Update Location Answer, then a subscriber can attain service only when it is currently located in a cell with TAC included in one of the HSS provided zone codes for the subscriber. Zone codes in Magma are configured at the network level utilizing the API end point `/lte/{network_id}/cellular/epc`. An example configuration is shown below:

Reviewer: ```suggestion
Magma also supports a zone code based service area restriction in federated LTE deployments. To utilize this service restriction, network operators need to configure the following

- An HSS with up to 10 Regional Restriction Zone Codes for each subscriber
- Magma with mapping between zone codes and lists of tracking area codes (TACs)

If the HSS does not provide any zone code in Update Location Answer for a subscriber, then no tracking area restriction is applied for the subscriber as the default behavior. On the other hand, if HSS provides a list of zone codes in Update Location Answer, then a subscriber can attain service only when it is currently located in a cell with TAC included in one of the HSS provided zone codes for the subscriber.

Zone codes in Magma are configured at the network level utilizing the API endpoint `/lte/{network_id}/cellular/epc`. An example configuration is shown below
```

> ```

Reviewer: Throughout this doc:
```suggestion
```json
```

{
  :
  :
  "service_area_maps": {
    "1234": [10, 20],
    "1235": [10, 30, 40],
    "1236": [20, 40, 50, 60]
  },
  :
  :
}

> ```

Reviewer: Add newline before and after code blocks

> Important thing to note here is that the zone code used as the key (e.g., "1234") is a string that corresponds to the decimal representation of the 2-octet zone code that is provided in the Regional-Restriction-Zone-Code AVP (e.g., `\x04D2`).

Reviewer: ```suggestion
An important thing to note here is that the zone code used as the key (e.g., `1234`) is a string that corresponds to the decimal representation of the 2-octet zone code that is provided in the Regional-Restriction-Zone-Code AVP (e.g., `\x04D2`).
```

> Service area restrictions can be added, updated, or removed via NMS using the JSON editor and adding, editing, or removing `service_area_maps` key according to the key hierarchy as below:
> ```

Reviewer: ```suggestion
Service area restrictions can be added, updated, or removed via NMS using the JSON editor and adding, editing, or removing `service_area_maps` key according to the key hierarchy as below

```
```
